app/Http/Controllers/ProfileController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
+
+
+class ProfileController extends Controller
+{
+    Public function index()
+    {
+        $user = Auth::user();
+
+        return view("auth.profile",
+        [
+            'user' => $user
+        ]);
+    }
+
+    public function edit(Request $request)
+    {
+        $user = Auth::user();
+
+        $validatedData = $request->validate([
```

Comment:
Dit is een behoorlijk lange array die er niet zo goed uit en duidelijk uit zit. Dit kan je verbeteren door een Request methode aan te maken. Plaats ook de validations in een array zoals 
```php
'first_name' => ['required', 'string', 'max:255']
```

Meer informatie over een Request class
[https://laravel.com/docs/11.x/validation#creating-form-requests](https://laravel.com/docs/11.x/validation#creating-form-requests)

Reply:
done dankjewel dit is echt handig